src/sentry/static/sentry/app/utils/discover/fieldRenderers.tsx

```diff
@@ -388,6 +390,48 @@ type SpecialFunctions = {
  * or they require custom UI formatting that can't be handled by the datatype formatters.
  */
 const SPECIAL_FUNCTIONS: SpecialFunctions = {
+  user_misery_prototype: data => {
+    const uniqueUsers = data.count_unique_user;
+    let miseryField: string = '';
+    let userMiseryField: string = '';
+    for (const field in data) {
+      if (field.startsWith('user_misery_prototype')) {
+        miseryField = field;
+      } else if (field.startsWith('user_misery')) {
+        userMiseryField = field;
+      }
+    }
+
+    if (!miseryField) {
+      return <NumberContainer>{emptyValue}</NumberContainer>;
+    }
+
+    const miserableUsers = userMiseryField ? data[userMiseryField] : undefined;
+
+    const userMisery = data[miseryField];
+    if (!uniqueUsers && uniqueUsers !== 0) {
+      return (
+        <NumberContainer>
+          {typeof userMisery === 'number' ? formatFloat(userMisery, 4) : emptyValue}
+        </NumberContainer>
+      );
+    }
+
+    const miseryLimit = parseInt(miseryField.split('_').pop() || '', 10);
```

Comment:
Good catch, I just had what the previous user misery was using, but looks like we can get organization for free in the baggage

Reply:
Im just looking at this again but I realize that this might not always work since discover allows the user to specify the threshold and it defaults to the `apdexThreshold`. Meaning that this parsing might be necessary 😞 sorry about that.

Reply:
Aah right, will work for performance, but not discover